client/src/constants/messages.js

```diff
@@ -21,6 +20,10 @@ export const TICKETS =
   'This page allows you to submit a help request via an RT Ticket.';
 export const UI =
   'This hidden page allows developers to review UI components in isolation.';
+
+/**
+ * Error Message Texts
```

Comment:
Would these message types be best described as text since the messages may include markup?

Reply:
Nope. How about "Error Messages"?

Reply:
I like it!

Reply:
[Updated.](https://github.com/TACC/Core-Portal/blob/quick/constant-messages-comment-update/client/src/constants/messages.js#L25)